src/app/cake/cake.service.ts

```diff
@@ -0,0 +1,29 @@
+import { ICake, CakeStatus } from '../models/index';
+
+const cakes: ICake[] = [
+  {
+    description: 'Tasty cake',
+    ingredients: ['Glass', 'Sugar'],
+    price: 12.99,
+    name: 'Red velvet',
+    status: CakeStatus.LastUnits,
+    stock: 3,
+  },
+  {
+    description: 'Tastier cake',
+    ingredients: ['Glass', 'Sugar', 'Egg'],
+    price: 13.99,
+    name: 'Cheese Cake',
+    status: CakeStatus.Available,
+    stock: 10,
+  },
+];
+
+export const getCakes = async (): Promise<ICake[]> => {
+  await timeout(2000);
+  return [...cakes];
```

Comment:
you can return the array directly: `return cakes`

Reply:
Solved